cdap-proto/src/main/java/co/cask/cdap/proto/audit/AuditMessage.java

```diff
@@ -26,6 +26,8 @@
  */
 @Beta
 public class AuditMessage {
+  private final int version = 1;
```

Comment:
Should it be a constant that users can access? For example, an app that processes AuditLog messages can have different logic based on the VERSION_1, VERSION_2 or something like that?